tagged-localbuild.xml

```diff
@@ -0,0 +1,825 @@
+  <project name="device-sony-lilac" path="device/sony/lilac" remote="sony" revision="789cecf51b3a59d4755f6e28153391c6de10a3bf" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
+  <project name="device-sony-maple" path="device/sony/maple" remote="sony" revision="03b647d06e4644c1009f0e56dd0df8cb70652410" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
+  <project name="device-sony-mermaid" path="device/sony/mermaid" remote="sony" revision="838a30b2eab4c794291b8c76c3c4ec54c6920000" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
+  <project name="device-sony-nile" path="device/sony/nile" remote="sony" revision="74765131c11a4ef8e715f6b0e5a5b76946a159e4" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
+  <project name="device-sony-pdx201" path="device/sony/pdx201" remote="sony" revision="809b6b8f9e6995249195a50333e69251a8682565" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
+  <project name="device-sony-pdx203" path="device/sony/pdx203" remote="sony" revision="82cea9194a290cf4c28732f5cf43d42c8c3c8709" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
+  <project name="device-sony-pdx206" path="device/sony/pdx206" remote="sony" revision="3a88eb11fdf27d825b5674e2d1d112cae9ee48ed" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
+  <project name="device-sony-pdx213" path="device/sony/pdx213" remote="sony" revision="47cb16d82a5cb8a91fcdacf313b1ecdd4ff9401d" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
+  <project name="device-sony-pioneer" path="device/sony/pioneer" remote="sony" revision="f5e43dffef0f7ce6c8d1f7c5effc9ff8edfbf272" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
+  <project name="device-sony-poplar" path="device/sony/poplar" remote="sony" revision="013d1bd25310dd9be08086ac03cac9714cd439cd" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
+  <project name="device-sony-seine" path="device/sony/seine" remote="sony" revision="71ff591a05cca1b079050b815df1ff747f246bfd" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
+  <project name="device-sony-sepolicy" path="device/sony/sepolicy" remote="sony" revision="2bea01538fc6b9c6dd220a6d610a798addab35fc" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
+  <project name="device-sony-tama" path="device/sony/tama" remote="sony" revision="879102f3231a0937c2a473afb333a25755bb384d" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
+  <project name="device-sony-voyager" path="device/sony/voyager" remote="sony" revision="674b0de906a8f9891b76e77f9b1ffc445b73e96f" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
+  <project name="device-sony-yoshino" path="device/sony/yoshino" remote="sony" revision="a3411de18dbec7ec530023a486cf41a1185bdccb" upstream="r-mr1" dest-branch="r-mr1" groups="device"/>
+  <project name="device/common" revision="6b398d7f47aae5d42d126c49f0879fa20d2ca93b" upstream="refs/tags/android-11.0.0_r43" dest-branch="refs/tags/android-11.0.0_r43" groups="pdk-cw-fs,pdk"/>
```

Comment:
This and several others below still say _r43

Reply:
Those were untracked by sony, removed them when updating Sony repos.